docs/user_manual.md

```diff
@@ -71,8 +71,12 @@ Register and unregister with `helios [un]register <hostname> <unqiue ID>`.
 ### List Agents
 
 List agents with `helios hosts [optional hostname pattern]`. If your agents have labels like
-`key=value` (see below on how to label agents), you can filter on labels with 
-`helios hosts -l key1=value1 -b key2=value2`.
+`key=value` (see below on how to label agents), you can use host selector expressions like
+`helios hosts -s key1=value1 -s key2!=value2 -s key3 in (value3a, value3b)`.
```

Comment:
nitpick but you have to quote the parenthesis

`helios hosts -s key1=value1 -s key2!=value2 -s key3 in (value3a, value3b)`